.github/workflows/pr-vrt.yml

```diff
@@ -21,78 +17,52 @@ permissions:
   actions: 'read'
 
 jobs:
-  web_components:
+  generate_vrt_screenshots:
     runs-on: ubuntu-latest
-    env:
-      pipelineId: '315'
-      pipelineName: 'fluent-ui_VRT_Pipeline_web-components'
     steps:
       - uses: actions/checkout@v4
         with:
           fetch-depth: 0
-      - name: Run and publish VR screenshot
-        uses: ./.github/actions/run-publish-vr-screenshot
-        with:
-          fluentVersion: webcomponents
-          vrTestPackageName: 'vr-tests-web-components'
-          vrTestPackagePath: 'apps/vr-tests-web-components'
-          locationPrefix: 'FluentUI-web-components'
-          locationPostfix: 'vrscreenshotwebcomponents'
-          clientName: 'fluentui-web-components-v3'
 
-  react_components:
-    runs-on: ubuntu-latest
-    env:
-      pipelineId: '311'
-      pipelineName: 'fluent-ui_VRT_Pipeline_v9'
-    steps:
-      - uses: actions/checkout@v4
+      - name: Derive appropriate SHAs for base and head for `nx affected` commands
+        uses: nrwl/nx-set-shas@v4
         with:
-          fetch-depth: 0
-      - name: Run and publish VR screenshot
-        uses: ./.github/actions/run-publish-vr-screenshot
-        with:
-          fluentVersion: v9
-          vrTestPackageName: 'vr-tests-react-components'
-          vrTestPackagePath: 'apps/vr-tests-react-components'
-          locationPrefix: 'fluentuiv9'
-          locationPostfix: 'vrscreenshotv9'
-          clientName: 'fluentuiv9'
+          main-branch-name: 'master'
 
-  react:
-    runs-on: ubuntu-latest
-    env:
-      pipelineId: '312'
-      pipelineName: 'fluent-ui_VRT_Pipeline_v8'
-    steps:
-      - uses: actions/checkout@v4
+      - uses: actions/setup-node@v4
         with:
-          fetch-depth: 0
-      - name: Run and publish VR screenshot
-        uses: ./.github/actions/run-publish-vr-screenshot
+          cache: 'yarn'
+          node-version: '20'
+
+      - run: yarn install --frozen-lockfile
+      - run: yarn playwright install --with-deps
+
+      - name: Run VR tests (generate screenshots)
+        run: yarn nx affected -t test-vr --nxBail
```

Comment:
this saves significant resources 

- before: install,playwright setup, scrernshot generation was done N times ( per affected projects ) 
- after: setup is invoked only once